models/biocro/R/met2model.BIOCRO.R

```diff
@@ -173,16 +173,16 @@ cf2biocro <- function(met, longitude = NULL, zulu2solarnoon = FALSE) {
   }
   
   ## Convert RH from percent to fraction BioCro functions just to confirm
-  if (met[, max(relative_humidity) > 1]) {
-    met[, `:=`(relative_humidity = relative_humidity/100)]
+  if (met[, max(met$relative_humidity) > 1]) {
+    met$relative_humidity = met$relative_humidity/100
```

Comment:
FWIW I think 'data.table' way was faster, as would the dplyr mutate function, though this is definitely more standard and readable.